src/Maker/MakeConvertPhpServices.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Bundle\MakerBundle\FileManager;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Util\PhpServicesCreator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+
+/**
+ * @author Antoine Michelet <jean.marcel.michelet@gmail.com>
+ */
+final class MakeConvertPhpServices extends AbstractMaker
+{
+    private $fileManager;
+
+    public function __construct(FileManager $fileManager)
+    {
+        $this->fileManager = $fileManager;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:convert-php-services';
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies)
+    {
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    {
+        $command->setDescription('Converts services.yaml file to services.php');
+
+        $command->addArgument('warning-comment',
```

Comment:
I'd call this `confirm`. And I'd make it an `option` so that you could pass `--confirm` when running the command (if you want to).

Then, create an `interact()` method - example https://github.com/symfony/maker-bundle/blob/b38c75be880b152ab55cef6cd52bf882d2b6518e/src/Maker/MakeResetPassword.php#L70 - and if the `confirm` option is not set, THEN you would ask the confirmation question using `$io->confirm()`.

Reply:
I need to take the time for learning how the command component works.

Reply:
Done.